ads/google/a4a/traffic-experiments.js

```diff
@@ -71,17 +76,22 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
     const isSetFromUrl = maybeSetExperimentFromUrl(win, element,
         experimentName, externalBranches.control,
         externalBranches.experiment, MANUAL_EXPERIMENT_ID);
-    const experimentInfo = {};
-    experimentInfo[experimentName] = internalBranches;
+    const experimentInfoMap = {};
+    experimentInfoMap[experimentName] = {
```

Comment:
The format change is reasonable, but why not just migrate the existing experiment info descriptions to the new format, rather than copying them here at runtime?